app/prisma/schema.prisma

```diff
@@ -200,8 +200,11 @@ model DatasetEntry {
     updatedAt DateTime @updatedAt
 }
 
+// TODO rename Organization to Project
 model Organization {
     id                String  @id @default(uuid()) @db.Uuid
+    name              String  @default("Project 1")
```

Comment:
Is there a good reason to set a default here? Most of the time "Project 1" won't be the right name, and I think it's better for us to get a typescript error if we create a project and forget to populate this field than have it have a misleading default.

Reply:
I think you're right, we don't really need a default name here. This was the simplest way I could think of to name all the existing organization records that don't currently have a name. What do you think a better approach is?

Reply:
I usually do this in 3 steps -- create the field as nullable, set the value for all existing entries, then add the non-null constraint. Eg. https://github.com/OpenPipe/OpenPipe/blob/main/app/prisma/migrations/20230718201303_add_users_and_orgs/migration.sql#L79-L86

You could also do it in two steps I suppose -- create the field with a default value, then remove the default value. Seems slightly less clear to me but either way should work.